api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -493,6 +496,33 @@ private void formatTimestampValues(List<DataSetPreviewValueList> valuePreviewLis
     return ResponseEntity.ok(response);
   }
 
+  @Override
+  public ResponseEntity<WgsCohortExtractionJob> extractCohortGenomes(
```

Comment:
should now be `extractDataSetGenomes`

Reply:
Updated model and method names per slack